### DISCUSSION TITLE
Apicurio registry mem VS jpa images

### DISCUSSION THREAD

**Discussion by teluren** [Apicurio/apicurio-registry#1859] [Q&A] [answered]
Hello Apicurio team.
Could you please give some information about the JPA image update on the docker hub?
We've installed the latest https://hub.docker.com/r/apicurio/apicurio-registry-jpa - but the v2 API doesn't work. Only http://{host}:{port}/api are available. 
With the https://hub.docker.com/r/apicurio/apicurio-registry-mem all's fine.
Regards, 
Peter.

**Comment by carlesarnal** [comment, marked as answer]
Hi @teluren, we've replaced the jpa image with the [SQL](https://hub.docker.com/r/apicurio/apicurio-registry-sql/) one. Could you give it a try?

**Reply by teluren** [reply]
Hi Carles,
Thanks a lot. We've used your new image, v2 API is working correctly.

**Reply by carlesarnal** [reply]
Glad it worked, thank you.
